app/src/main/java/com/hyperether/getgoing/db/DbHelper.java

```diff
@@ -14,9 +13,14 @@
  */
 public class DbHelper {
 
+    public interface Callback {
+        public void handlerDone();
```

Comment:
@NikolaHadzicNs give method some intuitive and also Android styled name, like onLoaded(); or similar